src/components/TopBar/AboutDialog.js

```diff
@@ -392,7 +408,62 @@ const Module = ({ type, name, version, license }) => {
                 },
             }}
         >
-            <Tooltip title={(name || '<?>') + ' ' + (version || '')}>
+            <Tooltip
+                TransitionComponent={Zoom}
+                enterDelay={2500}
```

Comment:
It's so slow... 1000 maximum

Reply:
That wanted, because it spam the view if you're juste scrolling in the list

Reply:
I could see 500 by the way...

Reply:
Better ask Stéphane about this